test/integration/ggrc/converters/test_export_csv.py

```diff
@@ -13,6 +13,7 @@
 from ggrc.integrations import constants
 from ggrc.models import inflector, all_models
 from ggrc.models.reflection import AttributeInfo
+from ggrc.snapshotter.rules import Types
```

Comment:
Please do not import classes directly as per style guide. Please rewrite it.
```suggestion
from ggrc.snapshotter import rules as snapshotter_rules
```